controllers/front/validation.php

```diff
@@ -119,101 +111,128 @@ public function postProcess()
             $this->displayExtError($blockonomics);
         }
 
-        // Create backup cart
-        $old_cart_secure_key = $cart->secure_key;
-        $old_cart_customer_id = (int)$cart->id_customer;
-        $cart_products = $cart->getProducts();
-        $new_cart = new Cart();
-        $new_cart->id_lang = $this->context->language->id;
-        $new_cart->id_currency = $this->context->currency->id;
-        $new_cart->add();
-        foreach ($cart_products as $product) {
-            $new_cart->
-                updateQty(
-                    (int) $product['quantity'],
-                    (int) $product['id_product'],
-                    (int) $product['id_product_attribute']
-                );
-        }
-        if ($this->context->cookie->id_guest) {
-            $guest = new Guest($this->context->cookie->id_guest);
-            $new_cart->mobile_theme = $guest->mobile_theme;
-        }
+        $sql = 'SELECT * FROM '._DB_PREFIX_."blockonomics_bitcoin_orders WHERE id_cart = $cart->id";
+        $order = Db::getInstance()->getRow($sql);
+        $time_left = $this->get_time_left($order);
```

Comment:
Let's use `time_remaining` instead of `time_left` to maintain consistency between platforms.

Reply:
Done.